rubocop/rubocop.gemspec

```diff
@@ -0,0 +1,19 @@
+# coding: utf-8
+# frozen_string_literal: true
+
+Gem::Specification.new do |spec|
+  spec.name = "rubocop-config-umbrellio"
+  spec.version = "0.48.1"
```

Comment:
Semantic versioning does not make sence here, i used model which is common when a gem requires exact version of another. If this will be improved by adding another version number, like '0.48.1.0', would it be ok?